contracts/libs/ParamManager.sol

```diff
@@ -1,69 +1,69 @@
 pragma solidity ^0.5.15;
 
 library ParamManager {
-    function DEPOSIT_MANAGER() public pure returns (bytes32) {
+    function depositManager() public pure returns (bytes32) {
         return keccak256("deposit_manager");
     }
 
-    function WITHDRAW_MANAGER() public pure returns (bytes32) {
+    function withdrawManager() public pure returns (bytes32) {
         return keccak256("withdraw_manager");
     }
 
-    function TEST_TOKEN() public pure returns (bytes32) {
+    function testToken() public pure returns (bytes32) {
         return keccak256("test_token");
     }
 
-    function POB() public pure returns (bytes32) {
+    function proofOfBurn() public pure returns (bytes32) {
         return keccak256("pob");
     }
 
-    function Governance() public pure returns (bytes32) {
+    function governance() public pure returns (bytes32) {
         return keccak256("governance");
     }
 
-    function ROLLUP_CORE() public pure returns (bytes32) {
+    function rollupCore() public pure returns (bytes32) {
         return keccak256("rollup_core");
     }
 
-    function ACCOUNT_REGISTRY() public pure returns (bytes32) {
+    function accountRegistry() public pure returns (bytes32) {
         return keccak256("account_registry");
     }
 
-    function LOGGER() public pure returns (bytes32) {
+    function logger() public pure returns (bytes32) {
         return keccak256("logger");
     }
 
-    function MERKLE_UTILS() public pure returns (bytes32) {
+    function merkleUtils() public pure returns (bytes32) {
         return keccak256("merkle_lib");
     }
 
-    function PARAM_MANAGER() public pure returns (bytes32) {
+    function paramManager() public pure returns (bytes32) {
         return keccak256("param_manager");
     }
 
-    function TOKEN_REGISTRY() public pure returns (bytes32) {
+    function tokenRegistry() public pure returns (bytes32) {
         return keccak256("token_registry");
     }
 
-    function TRANSFER() public pure returns (bytes32) {
+    function transferSimple() public pure returns (bytes32) {
```

Comment:
We don't use the name `transfer` because otherwise solhint would think we are doing a token transfer and warns reentry risk.